src/main/kotlin/no/nav/syfo/consumer/SykmeldingerConsumer.kt

```diff
@@ -0,0 +1,76 @@
+package no.nav.syfo.consumer
+
+import com.fasterxml.jackson.databind.DeserializationFeature
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule
+import com.fasterxml.jackson.module.kotlin.registerKotlinModule
+import io.ktor.client.*
+import io.ktor.client.call.*
+import io.ktor.client.engine.cio.*
+import io.ktor.client.features.json.*
+import io.ktor.client.request.*
+import io.ktor.client.statement.*
+import io.ktor.http.*
+import io.ktor.util.*
+import no.nav.syfo.Environment
+import no.nav.syfo.auth.StsConsumer
+import no.nav.syfo.consumer.syfosmregister.SyfosmregisterResponse
+import org.slf4j.LoggerFactory
+import java.time.LocalDate
+
+@KtorExperimentalAPI
+class SykmeldingerConsumer(env: Environment, stsConsumer: StsConsumer)  {
+
+    private val client: HttpClient
+    private val stsConsumer: StsConsumer
+    private val basepath: String
+    private val log = LoggerFactory.getLogger("no.nav.syfo.consumer.SyfosyketilfelleConsumer")
+
+    init {
+        client = HttpClient(CIO) {
+            install(JsonFeature) {
+                serializer = JacksonSerializer {
+                    registerKotlinModule()
+                    registerModule(JavaTimeModule())
+                    configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
+                }
+            }
+        }
+        this.stsConsumer = stsConsumer
+        basepath = env.syfosmregisterUrl
+    }
+
+    suspend fun getSykmeldingerForVarslingDato(dato: LocalDate, fnr: String): List<SyfosmregisterResponse>? {
+        val datoString = dato.toString()
+        val requestURL = "$basepath/api/v1/internal/sykmeldinger/?fom=/$datoString/?tom=/$datoString"//?include=SENDT/
+        log.info("[AKTIVITETSKRAV_VARSEL]: Syfosmregister requestURL: [$requestURL]")
+        val stsToken = stsConsumer.getToken()
+        val bearerTokenString = "Bearer ${stsToken.access_token}"
+
+        val response = client.get<HttpResponse>(requestURL) {
+            headers {
+                append(HttpHeaders.Authorization, bearerTokenString)
+                append(HttpHeaders.Accept, "application/json")
+                append("fnr", fnr)
+            }
+        }
+        log.info("[AKTIVITETSKRAV_VARSEL]: SyfosmregisterResponse, response: [$response]")
+        return when (response.status) {
+            HttpStatusCode.OK -> {
+                log.info("[AKTIVITETSKRAV_VARSEL]: SyfosmregisterResponse, response: [$response]")
+                response.receive<List<SyfosmregisterResponse>>()
+            }
+            HttpStatusCode.NoContent -> {
```

Comment:
Er dette (No Content) noe vi forventer at syfosmregister kan returnere? Ser at det er en Not Found respons type: 
https://github.com/navikt/syfosmregister/blob/a4fbcf29c30f6d2b2001de5d8c8dcf748c241056/src/main/kotlin/no/nav/syfo/sykmelding/serviceuser/api/SykmeldingServiceuserApi.kt#L26